### DISCUSSION TITLE
Previously used Isometric illustrations?

### DISCUSSION THREAD

**Discussion by thp44** [tailwindlabs/tailwindcss#7704] [Help] [answered]
Hi, 
Some time back tailwindcss/tailwindui/... used "technical isometric illustrations" I remember the illustration had some depth and colorful transparent layers. I can't find the name of the illustrations or the artist. Can someone help?

**Comment by adamwathan** [comment, marked as answer]
Hey! These are the ones you're looking for: https://www.pixsellz.io/lucid-illustrations

**Reply by thp44** [reply]
Yup, Thank you for the answer and for making Tailwindcss better every day! 🎉🎉🎉🎉 Keep up the awesome work, much appreciated! 👍
